src/vs/workbench/contrib/markers/browser/markersViewActions.ts

```diff
@@ -59,7 +59,7 @@ export class MarkersFilters extends Disposable {
 	set excludedFiles(filesExclude: boolean) {
 		if (this._excludedFiles.get() !== filesExclude) {
 			this._excludedFiles.set(filesExclude);
-			this._onDidChange.fire(<IMarkersFiltersChangeEvent>{ excludedFiles: true });
+			this._onDidChange.fire({ excludedFiles: true });
```

Comment:
In these cases, the function already has the right type for the argument so there's no need to have the type assertion